api-docs/v2/getting-started/install-CLI-client.rst

```diff
@@ -23,7 +23,7 @@ underneath your current directory.
       prod:
         auth:
           auth_url: https://identity.api.rackspacecloud.com/v2.0/
-          project_id: 123456
+          project_id: <RACKSPACE_CLOUD_TENANT_ID>
```

Comment:
I just checked our cross-product policy and we do want to use tenant id. This is great, thanks!